src/main/scala/com/hs/pipeline/demo/schema/Contract.scala

```diff
@@ -22,7 +24,8 @@ object Contract {
       Amount,
       Country,
       Entity,
-      OrderTime
+      OrderTime,
+    participation
```

Comment:
contract raw schema does not include a "participation" column, it's a calculated column, you can define it in the BV schema or somewhere else

Reply:
en fait BV ça refere à quoi ?